cypress/integration/dashboard_spec.js

```diff
@@ -1,3 +1,79 @@
 describe('The Dashboard Page', function () {
 
+    /* WHEN LOGGING IN */
+    describe('redirects to the dashboard', () => {
+        before(() => {
+            cy.request({
+                url: '/auth/login',
+                method: 'POST',
+                body: {
+                    username: 'unicorn'
+                }
+            });
+        })
+    
+        after(() => {
+            cy.request({
+                url: '/auth/logout',
+                method: 'POST'
+            });
+        })
+        it('successfully loads', function() {
```

Comment:
I don't think we should be testing this. If the page doesn't load, we have a bigger problem.